Cargo.lock

```diff
@@ -57,6 +57,21 @@ dependencies = [
  "memchr",
 ]
 
+[[package]]
```

Comment:
Oof, this is pulling in a lot of new third party deps! Can we use features to cut them back down? I'd naively assume that we were already pulling in the relevant bits of code for the parquet s3 sink

Reply:
Yeah good call, I disabled default features and it removed all of the new deps